test/test_file_processing.py

```diff
@@ -205,29 +207,61 @@ def add_tracks_and_tag(self, file_processing, recording):
         tag.id_ = file_processing.add_track_tag(track, tag)
         return track, tag
 
+    def process_all_recordings(self, helper, file_processing):
```

Comment:
Added this, so there should be only this test recordings as valid ones to be reprocessed.